slacksite/slackbot/views.py

```diff
@@ -11,6 +14,42 @@ def posting(request):
     user_id = r.get('user_id')
     user_name = r.get('user_name')
     text = r.get('text')
-    insert_user_db(user_id, user_name)
+    new_user, created = User.objects.get_or_create(user_id=user_id, 
```

Comment:
Do not use `new_user`. We don't know if the user already exists in database or not.